voluncheer/profiles/tests/test_views.py

```diff
@@ -74,3 +77,26 @@ def test_signup_form_exists_at_desired_location(self):
         """Tests accessing home by URL"""
         response = self.client.get("")
         self.assertEqual(response.status_code, 200)
+
+
+class OrganizationProfileTest(TestCase):
+    def setUp(self):
+        super().setUp()
+
+    def test_confirm_attandance(self):
+        rf = RequestFactory()
+        get_request = rf.get(
+            f"/attandance/{self.opp.pk}",
+            {"volunteer-attanded": [str(self.vol.pk)]},
+        )
+        get_request.user = self.org
+        confirm_attandance(get_request, self.opp.pk)
+        self.assertEqual(self.opp.attanded_volunteers.count(), 0)
+        self.opp.volunteers.add(self.vol)
+        self.opp.refresh_from_db()
+        confirm_attandance(get_request, self.opp.pk)
```

Comment:
There are three different assertions going on here based on different actions. Please provide some kind of separation to make it clear where a particular test/assertion is occurring. One good option is the use of subTest: https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests

Another thing to keep in mind when writing tests is the arrange, act, assert format. There are many posts on this topic, here is one as an example: https://learn.microsoft.com/en-us/visualstudio/test/unit-test-basics?view=vs-2022